iocore/aio/AIO.cc

```diff
@@ -587,7 +576,13 @@ DiskHandler::mainAIOEvent(int event, Event *e)
   }
 
   while ((op = complete_list.dequeue()) != nullptr) {
-    op->handleEvent(event, e);
+    op->mutex = op->action.mutex;
+    MUTEX_TRY_LOCK(lock, op->mutex, trigger_event->ethread);
+    if (!lock.is_locked()) {
+      trigger_event->ethread->schedule_imm(op);
```

Comment:
some delay here is better?

Reply:
Yes, it makes the DiskHandler running in non-blocked. 
In the original native AIO's `io_complete`, it always calls `SCOPE_MUTEX_LOCK` to acquire lock in blocked way.